ocaml/testsuite/tests/typing-modes/portable-contend.ml

```diff
@@ -46,17 +46,50 @@ let foo (r @ contended) = {r with b = best_bytes ()}
 Line 1, characters 27-28:
 1 | let foo (r @ contended) = {r with b = best_bytes ()}
                                ^
-Error: This value is contended but expected to be uncontended.
+Error: This value is contended but expected to be shared.
   Hint: In order to read from the mutable fields,
+  this record needs to be at least shared.
+|}]
+
+(* Writing to a mutable field in a shared record is rejected *)
+let foo (r @ shared) = r.a <- 42
+[%%expect{|
+Line 1, characters 23-24:
+1 | let foo (r @ shared) = r.a <- 42
+                           ^
+Error: This value is shared but expected to be uncontended.
+  Hint: In order to write into the mutable fields,
   this record needs to be uncontended.
 |}]
 
+(* Reading from a mutable field in a shared record is fine *)
```

Comment:
This comment doesn't seem to go with this test -- I don't see reading of a mutable field.